RibonDAO/app
# [PROD3-684] Daily and monthly tickets card business

# Description

- Daily and monthly card business to collect tickets
- Adjust on context

## Does this pull request close any open issues?

- Put here the issue that must be closed with this PR
- eg: closes [#PROD3-684]()

## Did you use styled-components?


## Type of change

Please delete options that are not relevant.


## How to test

- Test with user with business and try to collect tickets
- Test with user without business and see if appears the cards
- Test with user with club too
